src/app/recipes/recipes.component.ts

```diff
@@ -52,4 +52,8 @@ export class RecipesComponent {
 
     return this.recipes.filter(recipe => extendedFilteredCategories.every(id => recipe.categories.some(category => category.id == id)));
   }
+  
+  getImageUrl(recipe: Recipe) {
+    return '/assets/img/dishes/'+ recipe.imageName;
```

Comment:
It would be nice to move this hard coded `/assets/img/dishes/` to configuration, as it is prone to change

Reply:
Note: the same goes for the hard coded api urls in services

Reply:
Should be fixed now